x-pack/platform/plugins/private/transform/public/app/sections/create_transform/components/step_details/step_details_form.tsx

```diff
@@ -826,7 +826,7 @@ export const StepDetailsForm: FC<StepDetailsFormProps> = React.memo(
                     defaultMessage: 'Choose a maximum page search size.',
                   }
                 )}
-                isInvalid={!isTransformFrequencyValid}
```

Comment:
@elastic/ml-ui, please confirm that the value copied from `EuiFormRow` is relevant